utils/sw-ldap-user-sync

```diff
@@ -69,7 +69,7 @@ for uid in data['member']:
 
     try:
         (userdn, userdata) = directory.search_s(uid, ldap.SCOPE_SUBTREE, filter, attrs)[0]
-        if users.has_key(userdata["uid"][0]):
+        if userdata["uid"][0] in users:
```

Comment:
this is a dict https://github.com/uyuni-project/uyuni/pull/458/commits/d7a39e2ceb44026ed028f57ecf2389edf9b6fb94#diff-df3a719a76e2c83a19bcfe11ecad9b6eR54